.github/workflows/conformance.yml

```
@@ -0,0 +1,35 @@
+name: Conformance Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  conformance:
+    permissions:
+      # Needed to access the workflow's OIDC identity.
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@d171c3b028d844f2bf14e9fdec0c58114451e4bf
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+
+      - name: Build sigstore-java
+        uses: gradle/gradle-build-action@3fbe033aaae657f011f88f29be9e65ed26bd29ef # tag=v2.3.3
+        with:
+          arguments: clean :sigstore-conformance:build
+
+      - name: Unpack sigstore-java distribution
+        run: tar -xvf ${{ github.workspace }}/sigstore-conformance/build/distributions/sigstore-conformance-0.3.0.tar
```

Still need to figure out how I can do this in a way that doesn't hardcode the version number.

---

This isn't a fantastic solution, but maybe just a glob in a pinch?

```suggestion
        run: tar -xvf ${{ github.workspace }}/sigstore-conformance/build/distributions/sigstore-conformance-*.tar
```

(Assuming that there's only one archive matching that pattern at the end the build.)